presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1174,6 +1191,17 @@ private static void assertParsed(String input, Node expected, Node parsed)
         }
     }
 
+    private static void assertInvalidExpression(String expression, String expectedErrorMsgRegex)
+    {
+        try {
+            Expression result = SQL_PARSER.createExpression(expression);
+            fail(format("Expected to throw ParsingException for input:[" + expression + "], but got:" + result));
```

Comment:
Remove `String.format` because you didn't use it. Remove `[]`. Add space after `:`.